tests/isotope_qty_test.py

```diff
@@ -107,7 +107,10 @@ def test_isotopequantity_ref_atoms_rad(radioisotope, iq_kwargs):
         assert iq.ref_atoms == iq_kwargs['bq'] / radioisotope.decay_const
     else:
         assert iq.ref_atoms == (
-            iq_kwargs['uci'] * UCI_TO_BQ / radioisotope.decay_const)
+            iq_kwargs['uci'] * UCI_TO_BQ / radioisotope.decay_const /
+            N_AV * radioisotope.A / radioisotope.A * N_AV)
+            # Note the last part cancels, but is necessary to get equality with
+            # floating points
```

Comment:
This is so weird. Can you explain why this happens, and whether there is a way to avoid it?

Reply:
It's one of these weird floating point issues again. The way I do thing is that I calculate `bq` from `atoms`, `uci` from `bq`, `g` from `uci` and then `atoms` from `g`. We always start at the quantity that was provided. In this case `uci` was provided, which is used to calculate `g` which then is used to calculate `atoms`. We have to follow this "path" of derivation to get precisely the same value, which requires the weird conversion to `g` and back.

Reply:
It could be avoided by testing with `isclose`, but I would have to check what tolerances would be necessary.

Reply:
To be honest I think it would make more sense to test if `iq.quantity_at(quantity, date) == iq_kwargs[quantity]` here (which I think was not always true before and that is why we had the issue), but that seems sort of obvious so I left it.

Reply:
And we want to test that the conversion from one unit to the other are right.

Reply:
I added an `isclose` function, `numpy.isclose` or `math.isclose` both don't work with `uncertanties` as there is a test for `infinity` not supported for that module.